build.gradle.kts

```diff
@@ -183,10 +183,12 @@ subprojects {
             html.isEnabled = true
         }
 
-        retry {
-            failOnPassedAfterRetry.set(false)
-            maxFailures.set(5)
-            maxRetries.set(2)
+        if (System.getenv("CI") != null) {
```

Comment:
There are 3 retry blocks (1 for each test type), please apply it to all 3/maybe make a function (probably not worth it)

Reply:
oh yeh - ok